### ISSUE TITLE
what's about internatialization?

### ISSUE THREAD

**Issue by ghost** [jackrosenthal/threelayout#17]
I don't think, that programmers, who speak non-natively english, choice this layout instead of their own -  latin based - layouts.

**Comment by jackrosenthal** [comment]
This isn't one of the goals of `3l`. By supporting English-first, the layout can be highly optimized for English use.

There are other layouts with similar symbols layers for other languages (e.g., Neo v2, ADNW). These layouts work great for German, but are painful for typing English, as the letter and bi/tri-graph distributions for German are quite different than English. This really goes to show that a layout needs to be catered to the native language.

That being said, if you mostly write English and need support for some other symbols for occasional use of other latin alphabets, `3l` leaves many of the outer-edge keys undefined (top row, some of the keys above enter, etc.) that you can use to add support for occasional other symbols as needed.

tl;dr, if you need primary support for a language other than English, this layout is not for you. (but it can be if you need only secondary support)
